ee/packages/license/src/v2/bundles.ts

```diff
@@ -1,33 +1,11 @@
-import type { LicenseModule } from '@rocket.chat/core-typings';
+import { CoreModules, type LicenseModule } from '@rocket.chat/core-typings';
```

Comment:
Importing a non type from a package of types? This looks weird to me...

Reply:
That array is declared as a constant, which essentially means it is a readonly tuple: not a value to be changed and manipulated but rather to be referenced.

Doing it this way allows us to use the value during runtime to validate whether any given module name is a core module or not